security/crowdsec/+POST_INSTALL.post

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# the configuration file used in reconfigure (i.e. settings.json) may eventually
+# have credentials, so we create a directory to contain it -- the directory
+# permissions will be copied to the file while generating the jinja template.
+
+# shellcheck disable=SC2174
+mkdir -p -m 0700 /usr/local/etc/crowdsec/opnsense
+
+configctl crowdsec reconfigure
```

Comment:
not entirely sure this is a good idea, similar issue to when templates are left behind.. this can be problematic but in general it isn't

Reply:
As I remember, the (minor) problem is that if I don't create the directory, the files will be world readable. I don't know what they might hold in the future so I try to stay safe.

Reply:
for this we usually use a "setup.sh" which can also be used for /var MFS where /var directories need to be created on each boot (with correct permissions). There's a lot of examples in the ports tree.